src/app/containers/Image/helpers/srcSetHelper.js

```diff
@@ -0,0 +1,29 @@
+const createSrcset = (originCode, locator, originalImageWidth) => {
+  const resolutions = [240, 320, 480, 624, 800];
+  const ichefUrl = 'https://ichef.bbci.co.uk/news';
+  if (originCode === 'pips') {
+    return null;
+  }
+
+  const requiredResolutions = resolutions.filter(
+    resolution => resolution <= originalImageWidth,
+  );
+
+  const urls = requiredResolutions.map(
+    resolution =>
+      `${ichefUrl}/${resolution}/${originCode}/${locator} ${resolution}w`,
+  );
+
+  if (
```

Comment:
If you move this if to before line 12, you can add the value to the array before you construct the string, rather than creating a separate standalone link here. also means your not constructing the URL in two places